### PR TITLE
Issue459 solved

### PR DESCRIPTION
**Pull request by career-tokens**
image

### CODE REVIEW

**Review by JacobHeldt — CHANGES_REQUESTED** [review]
Thank you for working on this @career-tokens! Can you please make sure the header takes up the full with and is at the top of the page?

**Comment by career-tokens** [comment]
Tbh for a full screen web app I believe that would look very weird
 now I have allowed the user to apply his/her classname to apply custom style

File: src/components/layout/PageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          placeholder="Search study groups, tutors and learning paths"
+        />
+        <PiMagnifyingGlassBold size={32} className='text-magnifying_glass_color absolute right-2 w-[21px] h-[21px] my-auto top-0 bottom-0'/>
+      </div>
+      <div className="relative">
+        {!readAll && (
+          <svg width="32" height="33" viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M26.0001 24.5H6.00009C5.82506 24.4989 5.65338 24.452 5.5022 24.3638C5.35101 24.2756 5.22563 24.1492 5.13857 23.9974C5.05152 23.8456 5.00584 23.6735 5.0061 23.4985C5.00637 23.3235 5.05257 23.1516 5.14009 23C5.96384 21.575 7.00009 17.9762 7.00009 13.5C7.00009 11.1131 7.9483 8.82387 9.63613 7.13604C11.324 5.44821 13.6131 4.5 16.0001 4.5C18.387 4.5 20.6762 5.44821 22.364 7.13604C24.0519 8.82387 25.0001 11.1131 25.0001 13.5C25.0001 17.9775 26.0376 21.575 26.8626 23C26.9502 23.1518 26.9964 23.3239 26.9966 23.4991C26.9967 23.6744 26.9508 23.8466 26.8635 23.9985C26.7761 24.1504 26.6504 24.2767 26.4989 24.3647C26.3473 24.4527 26.1753 24.4994 26.0001 24.5Z" fill="#D3E4FF"/>
```
Comment (zaibutcooler) [inline src/components/layout/PageHeader.tsx:44]: have you thought about putting these svgs into separate files, components or even variables, not a good practice.

File: tailwind.config.ts [hunk 2]
```diff
@@ -147,7 +147,10 @@ const tailwindConfig = {
         background:"#f2f7ff",
         background_shade:"#d5d9e0",
         background_tint:"#f3f8ff",
-
+        header_background: "#f3f4f8",
```
Comment (zaibutcooler) [inline tailwind.config.ts:150]: don't understand why you need to add more variables into global tailwind.

File: src/app/check/page.tsx [hunk 3]
```diff
@@ -1,7 +1,14 @@
-import React from 'react'
+
+'use client'
```
Comment (zaibutcooler) [inline src/app/check/page.tsx:2]: better clean up the check page after testing. if someone's also working in this part, and push, we'll get merge conflict.

**Review by zaibutcooler — CHANGES_REQUESTED** [review]

**Review by JacobHeldt — APPROVED** [review]

**Pull request merged**